### PR TITLE
feature/disable-enable-source-rules

### LINKED ISSUE #5437: Add a way to enable/disable an authentication/administration rule.
It should be usefull to have a way to disable/enable an authentication/adminuistration rule.
Right now the only way to disable it so delete it , so you need to recreate it after.

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Allow rules in sources to be enabled or disabled

NEWS file entries
=======================
Enhancements
-------------
* Individual source rules can be disabled

Issue
=====
Fixes #5437

Delete branch after merge
=========================
NO

Checklist
=========
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
@satkunas Can you add the status=enabled/disabled to source rules in the Admin?
Thank you

**Comment by fdurand** [comment]
Works as expected.

File: addons/upgrade/to-10.1-rule-status.pl [hunk 1]
```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+to-10-filter_engines.pl
```
Comment (fdurand) [inline addons/upgrade/to-10.1-rule-status.pl]: not related

File: addons/upgrade/to-10.1-rule-status.pl [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+=head1 DESCRIPTION
+
+Upgrades the filter engines format
```
Comment (fdurand) [inline addons/upgrade/to-10.1-rule-status.pl]: wrong description

**Comment by extrafu** [comment]
@jrouzierinverse Do the requested changes by @fdurand

**Pull request merged**